src/views/lighttable.c

```diff
@@ -780,6 +780,20 @@ static gboolean zoom_out_callback(GtkAccelGroup *accel_group, GObject *accelerat
   return TRUE;
 }
 
+static gboolean zoom_max_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                                           GdkModifierType modifier, gpointer data)
+{
+  dt_view_lighttable_set_zoom(darktable.view_manager, 1);
+  return TRUE;
+}
+
+static gboolean zoom_min_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                                           GdkModifierType modifier, gpointer data)
+{
+  dt_view_lighttable_set_zoom(darktable.view_manager, 2 * DT_LIGHTTABLE_MAX_ZOOM);
```

Comment:
The max zoom is 25, why 2 times it ? The control at the bottom of the lighttable show max at 25.

And indeed ALT+ - set 50 thumbs per row.

Reply:
I agree. That wasn't introduced by this change but I'm happy to fix it.